tailwind.config.js

```diff
@@ -8,6 +8,13 @@ module.exports = {
     extend: {},
   },
   plugins: [require("daisyui")],
+  safelist: [
```

Comment:
The text color classes are now applied dynamically. To ensure that the colors are applied as expected in the browser, they have been added here to Tailwind's safe list.